ui/style.css

```diff
@@ -289,3 +289,16 @@ clamp > .small .roundable {
     opacity: 0;
   }
 }
+
+/**
+ * Colours for active action buttons
+ *
+ * TODO: How do we make this compatible with other themes?
```

Comment:
You could use the Palette Colors from libadwaita's [Named Colors](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/named-colors.html)

Reply:
I thought there was something like that, but couldn't find them. Hopefully something from the Gnome Palette will fit. And (IIRC) anyone who doesn't have those defined will fall back to black (at which point the filled vs outline icons become a clear differentiator!)